.github/workflows/unset-environment-new-cli.yml

```
@@ -55,34 +55,40 @@ jobs:
         TEST_MODE: true
     - shell: bash
       run: |
-        CPP_DB=${{ fromJson(steps.analysis.outputs.db-locations).cpp }}
+        CPP_DB="${{ fromJson(steps.analysis.outputs.db-locations).cpp }}"
         if [[ ! -d $CPP_DB ]] || [[ ! $CPP_DB == ${{ runner.temp }}/customDbLocation/* ]]; then
```

You should also quote the variable references here:

```suggestion
        if [[ ! -d "$CPP_DB" ]] || [[ ! "$CPP_DB" == "${{ runner.temp }}/customDbLocation/*" ]]; then
```

Also, below.

---

Just noticed this as well, should be done now.